tests/bin/run-e2e-CI.sh

```diff
@@ -1,11 +1,8 @@
 #!/usr/bin/env bash
-if [[ ${RUN_E2E} == 1 ]]; then
-
-	WP_SITE_URL="http://localhost:8080"
-
-	# Set base url to that of e2e test suite.
-	export BASE_URL="$WP_SITE_URL"
-
-	# Run the tests
-	npm test
-fi
+while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' localhost:8084/?page_id=4)" != "200" ]];
```

Comment:
Thanks for catching this! I added a loop to check if the container is built within 5 minutes. If not, the process will end with an error message. Right now on Travis, the container is being built in less than a minute but I decided to wait for max of 5 minutes just in case. It takes about 4 minutes to build on my local machine.